float32[] tab_points_X
float32[] tab_points_Y
uint8 size
